app/src/main/java/com/keylesspalace/tusky/fragment/NotificationsFragment.java

```diff
@@ -430,6 +431,24 @@ private void setFavovouriteForStatus(int position, Status status, boolean favour
         updateAdapter();
     }
 
+    public void onVoteInPoll(int position, @NonNull List<Integer> choices) {
+        final Notification notification = notifications.get(position).asRight();
+        final Status status = notification.getStatus();
+
+        timelineCases.voteInPoll(status, choices)
+                .observeOn(AndroidSchedulers.mainThread())
+                .as(autoDisposable(from(this)))
+                .subscribe(
+                        (newPoll) -> setVoteForPoll(position, newPoll),
+                        (t) -> Log.d(TAG,
+                                "Failed to vote in poll: " + status.getId(), t)
+                );
+    }
+
+    private void setVoteForPoll(int position, Poll poll) {
```

Comment:
Should it be the same or you intentionally left it out for now

Reply:
Intentionally left out, polls are not yet shown in notifications. I will add them in a separate PR.